assets/js/components/OverlayNotification/AnalyticsAndAdSenseAccountsDetectedAsLinkedOverlayNotification.js

```diff
@@ -187,13 +187,20 @@ export default function AnalyticsAndAdSenseAccountsDetectedAsLinkedOverlayNotifi
 
 		dismissNotification();
 
-		const widgetClass = '.googlesitekit-widget--adsenseTopEarningPagesGA4';
-
-		global.history.replaceState( {}, '', `#${ ANCHOR_ID_MONETIZATION }` );
-		global.scrollTo( {
-			top: getContextScrollTop( widgetClass, breakpoint ),
-			behavior: 'smooth',
-		} );
+		setTimeout( () => {
+			const widgetClass =
+				'.googlesitekit-widget--adsenseTopEarningPagesGA4';
+
+			global.history.replaceState(
+				{},
+				'',
+				`#${ ANCHOR_ID_MONETIZATION }`
+			);
+			global.scrollTo( {
+				top: getContextScrollTop( widgetClass, breakpoint ),
+				behavior: 'smooth',
+			} );
+		}, 50 );
```

Comment:
Note for Code Reviwer: Some browsers seem to disregard the `scrollTo` call when the DOM is repainted. I'm not entirely sure. I was seeing this behavior on Stable Chrome on Linux, but FF Developer Edition worked fine. Wrapping this in a `setTimeout` fixed the issue. Not sure if there's other proper way to solve this. Putting the dismiss call after the scroll didn't help either. Maybe we can try to listen for completion of scroll event before dismissing, ('scrollend' event) but not sure about the browser support on that. Let me know what you think.